frontend/src/components/style/ListCar.scss

```diff
@@ -0,0 +1,58 @@
+:root {
+  --secondColor: rgb(145, 145, 149);
+  --secondColorF: rgb(102, 102, 102);
+  --gradientColor1: #00aac6;
+  --gradientColor2: #00c1b7;
+  --gradientColor3: #00dc9f;
```

Comment:
You shouldn't add new colors in other files than `index.scss`, as they'll overwrite the main ones and it will prove quite difficult to debug